README.md

```diff
@@ -56,28 +59,113 @@ Options:
 * CI provided by CircleCI: [draft-annotations-api](https://circleci.com/gh/Financial-Times/draft-annotations-api)
 
 ## Service endpoints
 
 For a full description of API endpoints for the service, please see the [Open API specification](./api/api.yml).
 
-### GET
+### GET - Reading draft annotations from PAC
 
 Using curl:
 
 ```
-curl http://localhost:8080/draft/content/b7b871f6-8a89-11e4-8e24-00144feabdc0/annotations | jq
+curl http://localhost:8080/draft/content/{content-uuid}/annotations | jq
 ```
 
-Or using [httpie](https://github.com/jkbrzt/httpie):
+A GET request on this endpoint fetches the draft annotations for a specific piece of content by calling 
+[Generic RW Aurora](https://github.com/Financial-Times/generic-rw-aurora).
+In case the success, annotations are enriched with concept information by calling 
+[UPP Concept Search API](https://github.com/Financial-Times/concept-search-api).
+In case annotations are not available in PAC, 
+Draft Annotations API fetches published annotations by calling 
+[UPP Public Annotations API](https://github.com/Financial-Times/public-annotations-api).
+Fetching published annotations is part of the strategy for dynamic importing legacy annotations in PAC. 
 
+This is an example of response body:
 ```
-http GET http://localhost:8080/draft/content/b7b871f6-8a89-11e4-8e24-00144feabdc0/annotations
+[
+  {
+    "predicate": "http://www.ft.com/ontology/annotation/hasAuthor",
+    "id": "http://www.ft.com/thing/fd6734a1-3ae2-30f3-98a1-e373f8da8bf1",
+    "apiUrl": "http://api.ft.com/people/fd6734a1-3ae2-30f3-98a1-e373f8da8bf1",
+    "type": "http://www.ft.com/ontology/person/Person",
+    "prefLabel": "Emily Cadman",
+    "isFTAuthor": true,
+  },
+  {
+    "predicate": "http://www.ft.com/ontology/annotation/hasContributor",
+    "id": "http://www.ft.com/thing/5bd49568-6d7c-3c10-a5b0-2f3fd5974a6b",
+    "apiUrl": "http://api.ft.com/people/5bd49568-6d7c-3c10-a5b0-2f3fd5974a6b",
+    "type": "http://www.ft.com/ontology/person/Person",
+    "prefLabel": "Lisa Barrett",
+    "isFTAuthor": true,
+  },
+  {
+    "predicate": "http://www.ft.com/ontology/annotation/about",
+    "id": "http://www.ft.com/thing/d7de27f8-1633-3fcc-b308-c95a2ad7d1cd",
+    "apiUrl": "http://api.ft.com/things/d7de27f8-1633-3fcc-b308-c95a2ad7d1cd",
+    "type": "http://www.ft.com/ontology/Topic",
+    "prefLabel": "Global economic growth"
+  }
+]
 ```
 
-Currently, this endpoint is a proxy to the annotations available in UPP, so it returns a payload consistent to the UPP Public Annotations API.
+### PUT - Writing draft annotations from PAC
+
+Using curl:
+```
+curl -X PUT \
+  http://localhost:8080/drafts/content/{content-uuid}/annotations \
+  -d '[
+        {
+          "predicate": "http://www.ft.com/ontology/annotation/hasContributor",
+          "id": "http://www.ft.com/thing/5bd49568-6d7c-3c10-a5b0-2f3fd5974a6b",
+          "apiUrl": "http://api.ft.com/people/5bd49568-6d7c-3c10-a5b0-2f3fd5974a6b",
```

Comment:
Only the `predicate` and `id` fields are required.

Reply:
Clarified it the paragraph below to this snippet